### ISSUE TITLE
Bug in SatSL SatSV?

### ISSUE THREAD

**Issue by zlewko** [braamvandyk/SteamTables.jl#21]
I got an unexpected result from the Quality_Ps function and think there may be a scaling issue with SatSL and SatSV.  Their results appear to be 1000x what I would expect.

```
println( "Inputs")
@show P=0.101325        # 1 bar
@show T= Tsat(P)+1e-6 # shift up a bit for dry vapor
@show s = SpecificS(P,T)
println( "Quality_Ps Functions Calculation")
@show T = Tsat(P)
@show sl = SatSL(T)
@show sv = SatSV(T)
@show Quality_Ps_calc = (s - sl)/(sv - sl)
```

Gives:
Inputs
P = 0.101325 = 0.101325
T = Tsat(P) + 1.0e-6 = 373.12430100048056
s = SpecificS(P, T) = 7.354385119195204
Quality_Ps Functions Calculation
T = Tsat(P) = 373.12430000048056
sl = SatSL(T) = 1306.9121749247986
sv = SatSV(T) = 7354.9380255262795
Quality_Ps_calc = (s - sl) / (sv - sl) = -0.21487305476321025

**Comment by braamvandyk** [comment]
Yes, that does seem off by quite a bit. There is also an inaccuracy in the Tsat by about 0.03K. I'll have to go check if this is an implementation error or a problem with the source formula. Thanks for reporting this.

**Comment by braamvandyk** [comment]
The error was a unit inconsistency between the two source documents that I failed to notice. 

```
julia> P = 0.101325
0.101325

julia> T = Tsat(P) + 1e-6
373.12430100048056

julia> s = SpecificS(P, T)
7.354385119195204

julia> T = Tsat(P)
373.12430000048056

julia> sl = SatSL(T)
1.3069121749247985

julia> sv = SatSV(T)
7.3549380255262795

julia> Quality_Ps_calc = (s - sl)/(sv - sl)
0.9999085806931495
```
The remaining small error in the quality is due to the small error in Tsat, which is part and parcel of the source formula.

I'll push the new version as soon as I have double checked the last few things.

**Comment by zlewko** [comment]
Thanks.  I thought it was probably a g vs kg issue somewhere between two
different sources.  Today I tried to work around it and I noticed a similar
issue with the SatHV and SatLV functions, they appeared 1000x too high.

**Comment by braamvandyk** [comment]
Should be fixed now in v1.4.1 that I just tagged. Wait for registration in Julie registry,
